src/modules/endpoint/sagas/loadLocalNews.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import type { Saga } from 'redux-saga'
+import { createLocalNewsEndpoint, LocalNewsModel } from '@integreat-app/integreat-api-client'
+import { call } from 'redux-saga/effects'
+import determineApiUrl from '../determineApiUrl'
+
+function * loadLocalNews (
+  city: string,
+  language: string
+): Saga<Array<LocalNewsModel>> {
+  console.debug('Fetching news')
+
+  const apiUrl = yield call(determineApiUrl)
+  const payload = yield call(() => createLocalNewsEndpoint(apiUrl).request({ city, language }))
+
+  const newsList: Array<LocalNewsModel> = payload.data
```

Comment:
```suggestion
  const news: Array<LocalNewsModel> = payload.data
```
If you want to show that there are multiple news, I'd use `newsModels` instead.

Reply:
I think I will make it news to keep all naming consistent.